schema.json

```diff
@@ -202,6 +202,80 @@
             },
             "type": "object"
         },
+        "persist": {
+            "anyOf": [
+                {
+                    "description": "Deprecated, use object representation instead.",
+                    "type": "string"
+                },
+                {
+                    "description": "Deprecated, use object representation instead.",
+                    "items": {
+                        "$ref": "#/definitions/stringOrArrayOfStrings"
+                    },
+                    "minItems": 1,
+                    "type": "array"
+                },
+                {
+                    "minItems": 1,
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": [
+                            "name",
+                            "type"
+                        ],
+                        "if": {
+                            "properties": {
+                                "type": {
+                                    "const": "file"
+                                }
+                            }
+                        },
+                        "then": {
+                            "required": [
```

Comment:
These two properties should not be required, they could be ommited in most instances with no contents and encoding "UTF-8" (which is default in Windows 10 in despite of encoding with Default, UTF8, ASCII, OEM in `Out-File`).

If app require different encoding, we can set through `Get-Content | Out-File -Encoding`, but it should be rare case.